commands/plugin_e2e_test.go

```diff
@@ -0,0 +1,147 @@
+package commands
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestPluginEnableCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	pluginID := "com.mattermost.demo-plugin"
+	pluginURL := filepath.Join(os.Getenv("MM_SERVER_PATH"), "tests", "testplugin.tar.gz")
+	nonExistentPluginID := "nonExistentPluginID"
+
+	installPlugin(s, pluginID, pluginURL)
+	defer removePlugin(s, pluginID)
+
+	s.RunForSystemAdminAndLocal("Successful enable plugin", func(c client.Client) {
+		printer.Clean()
+
+		appErr := s.th.App.DisablePlugin(pluginID)
+
+		plugins, appErr := s.th.App.GetPlugins()
+		s.Require().Nil(appErr)
+		s.Require().Len(plugins.Active, 0)
+		s.Require().Len(plugins.Inactive, 1)
```

Comment:
I wonder if we ever need to check these? We are not testing if `DiesablePlugin` is working correctly.